### PR TITLE
chore(locales): Update english locales for storage, user, role, clust…

### PR DESCRIPTION
**Pull request by rayzhou2017**
…er, etc.

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

File: src/locales/zh/role.js [hunk 1]
```diff
@@ -185,7 +185,7 @@ export default {
   STORAGECLASSES_VIEW_DESC: '查看集群所有存储类型',
   STORAGECLASSES_MANAGEMENT_DESC: '创建/编辑/删除 存储类型, 设置默认存储类型',
   'Volume Snapshots View': '存储快照查看',
-  'Volume Snapshots Management': '存储快照管理',
+  'Volume Snapshots': '存储卷快照',
```
Comment (leoendless) [inline src/locales/zh/role.js:188]: This word has been defined in cluster role templates. It is a specific key.

File: src/locales/zh/cluster.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ export default {
   'Custom Resources': '自定义资源 CRD',
   'Storage Management': '存储管理',
   'Cluster Settings': '集群设置',
-  'Snapshots Management': '快照管理',
+  'Snapshots': '存储卷快照',
```
Comment (leoendless) [inline src/locales/zh/cluster.js:36]: need to change the title of ``volume-snapshots`` in ``config.yaml`` too.

File: src/locales/zh/network.js [hunk 3]
```diff
@@ -55,9 +55,9 @@ export default {
   NETWORK_POLICY_R2_TITLE: '集群外部IP地址',
   NETWORK_POLICY_R2_DESC: '允许集群外部CIRD访问',
   NETWORK_POLICY_R2_DESC1:
-    '选择特定的IP CIDR范围以允许作为入口源或出口目的地。集群外部IP',
-  NETWORK_POLICY_D_DESC: '匹配Egress(出口)流量以及Ingress(入口)流量',
-  NETWORK_POLICY_D_DESC2: 'CIDR代表一个合法的IP地址块,例如“192.168.1.1/24”',
+    '选择特定的 IP CIDR 范围以允许作为入口源或出口目的地。',
```
Comment (Sherlock113) [inline src/locales/zh/network.js:58]: I will delete "Cluster external IP" in en loc as well.

File: src/locales/zh/role.js (hunk 1 above)
Comment (rayzhou2017) [reply src/locales/zh/role.js:188]: I didn't search the key. Where is the cluster role template?

File: src/locales/zh/cluster.js (hunk 2 above)
Comment (rayzhou2017) [reply src/locales/zh/cluster.js:36]: Already changed in this commit

Comment (rayzhou2017) [reply src/locales/zh/cluster.js:36]: my mistake, I forgot to save the change.

Comment (rayzhou2017) [reply src/locales/zh/cluster.js:36]: I rebased the changes

File: src/locales/zh/role.js (hunk 1 above)
Comment (leoendless) [reply src/locales/zh/role.js:188]: It's in backend service.  
And I think there is no need to modify it. 'Volume Snapshots View' and 'Volume Snapshots Management' are permission names. Users can clearly identify the difference between these two permissions.

Comment (rayzhou2017) [reply src/locales/zh/role.js]: changed back

**Pull request merged**